string carSn
Position[] way_point
float64 yaw